providers/defltprov.c

```diff
@@ -527,6 +527,17 @@ static const OSSL_ALGORITHM deflt_serializer[] = {
     { "EC", "provider=default,fips=yes,format=pem,type=parameters",
       ec_param_pem_serializer_functions },
 #endif
+
+    { NULL, NULL, NULL }
+};
+
+static const OSSL_ALGORITHM deflt_deserializer[] = {
+    { "RSA", "provider=default,fips=yes,input=der",
```

Comment:
Add RSA_PSS?

Reply:
Yes

Reply:
Er, that will have to wait...  it seems that we don't have an RSA-PSS serializer???

Reply:
Are they not exactly the same?